### PR TITLE
** UI Redesign **

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR contains our work for the redesigned UI to be released with Aqueduct version 0.1.0.

## Related issue number (if any)
A lot of issues are covered here, will update with a detailed change list.

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/DataPreviewer/index.tsx [hunk 1]
```diff
@@ -102,6 +103,11 @@ const DataPreviewer: React.FC<Props> = ({
             sx={{
               position: 'absolute',
               height: dataTableHeight ? dataTableHeight : '100%',
+              // height: '-webkit-fill-available',
```
Comment (agiron123) [inline src/ui/common/src/components/DataPreviewer/index.tsx:106]: remove comments

**Review by likawind — COMMENTED** [review]
Let's remove the `package-lock.json` in `/app` based on https://aqueducthq.slack.com/archives/C01KF131001/p1665729295422109?thread_ts=1665720394.690999&cid=C01KF131001 . I also encountered this issue and it likely.  I'll note a few `console.log`s. But other than those, we should be good to go!

File: src/ui/common/src/components/pages/metric/id/index.tsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  const pathPrefix = getPathPrefix();
+  const workflowLink = `${pathPrefix}/workflow/${workflowId}?workflowDagResultId=${workflowDagResultId}`;
+  const breadcrumbs = [
+    BreadcrumbLink.HOME,
+    BreadcrumbLink.WORKFLOWS,
+    new BreadcrumbLink(workflowLink, workflow.selectedDag?.metadata.name),
+    new BreadcrumbLink(path, operator ? operator.name : 'Metric'),
+  ];
+
+  console.log('selected dag is', workflow.selectedDag);
```
Comment (likawind) [inline src/ui/common/src/components/pages/metric/id/index.tsx:83]: remove

**Review by likawind — COMMENTED** [review]
build failure caused by linter, let's run it again

**Comment by agiron123** [comment]
> build failure caused by linter, let's run it again

@likawind going to run it and push up here.

**Comment by likawind** [comment]
everything is great from my end

**Review by likawind — APPROVED** [review]
S6RB77Z

File: src/ui/app/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint '*/**/*.{js,ts,tsx}' --format table --fix"
   },
   "dependencies": {
-    "@aqueducthq/common": "0.0.19",
+    "@aqueducthq/common": "0.0.19.alpha",
```
Comment (agiron123) [inline src/ui/app/package.json:13]: before release we are going to want to change this to 0.1.0 to be in sync with the python package. Just want to call that out so we don't forget.

Same thing needs to happen with common of course.

**Pull request merged**